test/functions/compilation-errors/value-for-arg-type.chpl

```diff
@@ -0,0 +1,10 @@
+
+var myVar: int;
+proc p1(arg: myVar) { }
+p1(5);
+
+proc p2(arg: 1+2) { }
+p2(6);
+
+proc p3(argxx: Locales.size) { }
```

Comment:
Can we have another test case in here for a function with parens? Out of curiosity, how about a param variable as well?

Reply:
I added a function with parens. The param case is tested by `proc p2(arg: 1+2)`.
Good suggestions, thanks!